nvflare/fuel/hci/client/api.py

```diff
@@ -367,6 +422,9 @@ def do_command(self, command):
         return self.server_execute(command)
 
     def server_execute(self, command, reply_processor=None):
+        if not self.api_active:
+            return {"status": APIStatus.ERROR_INACTIVE_SESSION, "details": "API session is inactive"}
```

Comment:
I think the name `api_active` should be changed for clarity.
Since this actually means if the server session is active or not.

For example, the client side commands can actually run without issues even if server session is inactive.

Reply:
Sure, how about `server_session_active`